<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>223</width>
    <height>359</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>223</width>
    <height>359</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>223</width>
    <height>359</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pad_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pad_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pad_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pad_minus">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pad_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pad_multipy">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pad_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pad_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pad_9">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pad_equal">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pad_divide">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>รท</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pad_0">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>280</y>
      <width>101</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pad_8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pad_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pad_add">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pad_clear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QLabel" name="main_display">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    background-color: white;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="operation_display">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    background-color: white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pad_decimal">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>51</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pad_plus_minus">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>51</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>223</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
